challenge.js

```diff
@@ -42,7 +45,7 @@
  */
 
 // ANSWER
-
+// You can use the element selector.
```

Comment:
We were looking for the specific selector - one example might be `body > div > p`. Half credit on this one.